### PR TITLE
Training react

### PR DESCRIPTION
**Pull request by andyOlarte514**
# Summary
- add folder `finger-exercises` and run test of `01 - spread`, `02 - promises` in folder `ecmascript`

# Trello Card
- https://trello.com/c/FwOpueoK

### CODE REVIEW

File: training-react/finger-exercises/ecmascript/01 - spread/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { isArray } from './utils';
+
+export function min(param, ...arg) {
+  let val;
+  if (param !== undefined && param !== null) {
```
Comment (henryzarza) [inline training-react/finger-exercises/ecmascript/01 - spread/index.js]: You can replace this condition for `if (param)` see this [post](https://www.sitepoint.com/javascript-truthy-falsy/)

File: training-react/finger-exercises/ecmascript/01 - spread/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    if (isArray(param)) {
+      val = Math.min(...param);
+    } else if (arg.length > 0) {
```
Comment (henryzarza) [inline training-react/finger-exercises/ecmascript/01 - spread/index.js]: Same here, you can replace for `arg.length`

File: training-react/finger-exercises/ecmascript/01 - spread/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      arg.push(param);
+      val = Math.min(...arg);
+    } else {
+      val = param;
+    }
+  }
+  return val;
+}
+
+export function copy(data) {
```
Comment (henryzarza) [inline training-react/finger-exercises/ecmascript/01 - spread/index.js:18]: You can replace this for `return isArray(data) ? [...data] : { ...data }`

File: training-react/finger-exercises/ecmascript/01 - spread/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  let copyArr;
+  if (isArray(data)) {
+    copyArr = [...data];
+  } else {
+    copyArr = { ...data };
+  }
+  return copyArr;
+}
+
+export function reverseMerge(arr1, arr2) {
```
Comment (henryzarza) [inline training-react/finger-exercises/ecmascript/01 - spread/index.js:22]: You can replace the `push` function for `concat` function, see this [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good work! Only a few comments

**Review by henryzarza — DISMISSED** [review]
Good work 👍

File: training-react/finger-exercises/ecmascript/01 - spread/index.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  return val;
+}
+
+export function copy(data) {
+  return isArray(data) ? [...data] : { ...data };
+}
+
+export function reverseMerge(arr1, arr2) {
+  const newArr = [].concat(...arr2).concat(...arr1);
+  return newArr;
```
Comment (wfolini) [inline training-react/finger-exercises/ecmascript/01 - spread/index.js]: U can return `[...arr2, ...arr1]`

**Review by wfolini — CHANGES_REQUESTED** [review]
Great! Min change and go!

**Review by wfolini — APPROVED** [review]

**Pull request merged**